app/src/lib/stores/app-store.ts

```diff
@@ -1607,13 +1608,62 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
       return { workingDirectory, selectedFileIDs, diff }
     })
+
+    this._triggerMergeConflictsFlow(repository, status)
+
     this.emitUpdate()
 
     this.updateChangesDiffForCurrentSelection(repository)
 
     return true
   }
 
+  /** starts the conflict resolution flow, if appropriate */
+  private async _triggerMergeConflictsFlow(
+    repository: Repository,
+    status: IStatusResult
+  ) {
+    const inConflictedMerge = status.workingDirectory.files.some(f => {
+      return (
+        f.status === AppFileStatus.Conflicted ||
+        f.status === AppFileStatus.Resolved
+      )
+    })
+    if (!inConflictedMerge) {
+      return
+    }
+    if (status.currentBranch === undefined) {
+      return
+    }
+    // if we're already in the conflict merge resolution flow, bail
+    if (
+      this.currentPopup !== null &&
+      this.currentPopup.type !== PopupType.MergeConflicts &&
```

Comment:
This check seems to be for if we're not seeing the `MergeConflicts` popup and also not seeing in the `AbortMerge` popup - which cannot both be true - so the code below runs whenever `AppStore._loadStatus` fires.

I gather that's what the comment above is for, but it'd be great if we were expressive enough that we didn't need comments, like this:

```ts
if (this.currentPopup === null) {
  return
}

const mergeConflictFlowActive =
  this.currentPopup.type === PopupType.MergeConflicts ||
  this.currentPopup.type === PopupType.AbortMerge

if (mergeConflictFlowActive) {
  return
}
```